packages/documentation/pages/usage/components/field-text-area.vue

```diff
@@ -0,0 +1,63 @@
+<template lang="md">
```

Comment:
Not needed, instead just add the new props for KtFieldTextArea in `packages/documentation/pages/usage/components/form-fields.vue`